src/main.rs

```diff
@@ -89,6 +91,15 @@ fn main() -> Result<()> {
         .help("the project to which the new activity belongs")
         .takes_value(true);
 
+    let arg_format = Arg::with_name("format")
+        .short("o")
+        .long("output")
```

Comment:
In my experience "output" is more frequently used for where the outputted data should go.
I would suggest using `--format` as the long form and maybe `-F` as the shorthand.

Or make it a boolean flag and just do `--json`/`-j` (seen in `hyprctl` from [hyprland](https://github.com/hyprwm/Hyprland)).
I would personally prefer the latter (if more formats should come up in the future it could still be changed. Although I would say that if it is likely that there should be more formats supported, the concrete input is probably better as it would be less of a breaking change).